llvm/docs/LangRef.rst

```diff
@@ -727,7 +727,7 @@ optimizations based on the 'constantness' are valid for the translation
 units that do not include the definition.
 
 As SSA values, global variables define pointer values that are in scope
-(i.e. they dominate) all basic blocks in the program. Global variables
+for (i.e. they dominate) all basic blocks in the program. Global variables
```

Comment:
We should be able to read this sentence without the parenthesised words, but we can't without adding `for`.